scanner/decisions/0004-scannerv4-read-vulneralbilities.md

```diff
@@ -0,0 +1,41 @@
+# 0004 - Scanner V4 NVD CVSS enrichment data updating pipeline
+
+- **Author(s):** Yi Li
+- **Created:** [2023-09-11]
+
+## Context
+
+We are currently transitioning the StackRox Scanner to use ClairCore as its primary scanning engine. Our goals are twofold:
+
+1. Enable Central to retrieve NVD CVSS data and consolidate all CVSS data that contains available CVSS v3 scores.
+2. Ensure seamless integration between Scanner V4 and Central, for enriching vulnerabilities with CVSS data.
+
+Throughout this workflow, a new CVSS Updater equipped by a novel enricher in Central will download and consolidate the CVSS data, then creating a JSON bundle. 
+
+ClairCore will be instrumental in scanner V4 data retriever (will be explained in detail below) while requesting this CVSS data bundle from central, handling tasks such as downloading, parsing, and updating CVSS data for Scanner V4.
+
+## Decision
```

Comment:
```suggestion

## Decision

Central will have a new NVD CVSS Updater equipped by a novel enricher, based on ClairCore. It will download and consolidate the NVD CVSS data in a JSON bundle. 

ClairCore will also be used in Scanner V4 CVSS data retriever (will be explained in detail below). It will require the NVD CVSS data bundle stored in Central. Central will handle the task of downloading, parsing, and updating the NVD CVSS data bundle for Scanner V4 using ClairCore.
```